Documentation/ceph-examples.md

```diff
@@ -43,27 +41,37 @@ Settings for the operator are configured through environment variables on the op
 Now that your operator is running, let's create your Ceph storage cluster:
 - `cluster.yaml`: Common settings for a production storage cluster. Requires at least three nodes.
 - `cluster-test.yaml`: Settings for a test cluster where redundancy is not configured. Requires only a single node.
-- `cluster-minimal.yaml`: Brings up a cluster with only one mon and a mgr so the Ceph dashboard can be used for the remaining cluster configuration.
+- `cluster-minimal.yaml`: Brings up a cluster with only one [ceph-mon](http://docs.ceph.com/docs/mimic/man/8/ceph-mon/) and a [ceph-mgr](http://docs.ceph.com/docs/mimic/mgr/) so the Ceph dashboard can be used for the remaining cluster configuration.
 
 See the [Cluster CRD](ceph-cluster-crd.md) topic for more details on the settings.
 
-## Storage Class
-The storage class is defined with a pool which defines the level of data redundancy:
-- `storageclass.yaml`: Replication of 3 for production scenarios. Requires at least three nodes.
-- `storageclass-ec.yaml`: Configures erasure coding for data durability rather than replication. See the [Erasure coding](ceph-pool-crd.md#erasure-coded) documentation for more details. Requires at least three nodes.
-- `storageclass-test.yaml`: Replication of 1 for test scenarios. Requires only a single node.
+## Setting up consummable storage
+
+Now we are ready to setup [block](https://ceph.com/ceph-storage/block-storage/), [shared filesystem](https://ceph.com/ceph-storage/file-system/) or [object storage](https://ceph.com/ceph-storage/object-storage/) in the rook ceph cluster. These kinds of storage are respectively referred to as CephBlockPool, CephFilesystem and CephObjectStore in the spec files.
+### Block Devices
+
+Ceph can provide raw block device volumes to pods. Each example below sets up a storage class which can then be used to provision a block device in kubernetes pods. The storage class is defined with [a pool](http://docs.ceph.com/docs/jewel/rados/operations/pools/) which defines the level of data redundancy in ceph:
```

Comment:
I'd suggest we link to ceph-pool-crd.md instead of the ceph topic

Reply:
We do have the link to ceph-pool-crd.md at the end of this section
...
"See the Cluster CRD topic for more details on the settings."

Reply:
ok sounds good, if you could just update this link to the nautilus docs.